packages/Webkul/Shop/src/Resources/lang/ru/app.php

```diff
@@ -201,7 +201,7 @@
                         'subtotal'           => 'Промежуточная сумма',
                         'shipping-handling'  => 'Доставка и обработка',
                         'tax-amount'         => 'Сумма налога',
-                        'tax'                => 'Налог',
+                        'tax'               => 'Налог',
```

Comment:
Kindly add one space before the (=>) symbol.

Reply:
One space Added.